stock_inventory_valuation_report/reports/stock_inventory_valuation_report.xml

```diff
@@ -113,7 +115,13 @@
                 <t t-esc="n"/>
             </div>
             <div class="act_as_cell left">
-                <t t-esc="line.display_name"/>
+                <t t-esc="line.reference"/>
+            </div>
+            <div class="act_as_cell left">
+                <t t-esc="line.name"/>
+            </div>
+            <div class="act_as_cell left">
+                <t t-esc="line.barcode"/>
             </div>
```

Comment:
I think this report should show barcode image. Could you please do this?

Reply:
I tried but it doesn't look good. Either we use big rows, or the barcode is too small.
Anyways, who will use it to scan? It's an inventory valuation, barcode is usually used for reference in that case.

Our customer's only use case is to show the barcode numbers, not the picture

![image](https://user-images.githubusercontent.com/1914185/76229314-4983b580-6222-11ea-8afb-b602bb298ae3.png)

Reply:
Ok, I agree on it. So this report could show the barcode numbers.